From: Jari Aalto <jari.aalto@cante.net>
Subject: Correct spelling.

---
 ReadMe.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/ReadMe.c
+++ b/ReadMe.c
@@ -152,6 +152,6 @@
 "\n"
 "The 'browse' function provides an interactive mode for browsing a\n"
 "set of patches.  It allows the application of a patch to each file \n"
-"to be inspected and allows limited editting to correct mis-application\n"
+"to be inspected and allows limited editing to correct mis-application\n"
 "of patches where wiggling was required, and where conflicts occurred.\n"
 "\n";
