From 50fb6002ee57e69f38f17314acf22855066127e2 Mon Sep 17 00:00:00 2001
From: Jari Aalto <jari.aalto@cante.net>
Date: Thu, 4 Mar 2010 10:02:14 +0200
Subject: [PATCH] Debian 0.6-6 changes in code (author unknown)

Signed-off-by: Jari Aalto <jari.aalto@cante.net>
---
 bestmatch.c |    5 ++++-
 extract.c   |    8 ++++----
 merge.c     |    9 +++++++++
 3 files changed, 17 insertions(+), 5 deletions(-)

diff --git a/bestmatch.c b/bestmatch.c
index 5bb31ef..3d73577 100644
--- a/bestmatch.c
+++ b/bestmatch.c
@@ -346,7 +346,10 @@ void remap(struct best *best, int cnt,
 	int pa,pb;
 	pa=pb=0;
 
-	for (b=1; b<cnt; b++) {
+	if (a1.elcnt == 0 && a2.elcnt == 0) return;
+
+	for (b=1; b<cnt; b++)
+	   if (best[b].val>0) {
 #if 0
 		printf("best %d,%d  %d,%d\n",
 		       best[b].xlo,best[b].ylo,
diff --git a/extract.c b/extract.c
index 0959b44..755a725 100644
--- a/extract.c
+++ b/extract.c
@@ -207,7 +207,7 @@ int split_merge(struct stream f, struct stream *f1, struct stream *f2, struct st
 		lineno++;
 		switch(state) {
 		case 0:
-			if (len>8 &&
+			if (len>=8 &&
 			    strncmp(cp, "<<<<<<<", 7)==0 &&
 			    (cp[7] == ' ' || cp[7] == '\n')
 				) {
@@ -222,7 +222,7 @@ int split_merge(struct stream f, struct stream *f1, struct stream *f2, struct st
 			}
 			break;
 		case 1:
-			if (len>8 &&
+			if (len>=8 &&
 			    strncmp(cp, "|||||||", 7)==0 &&
 			    (cp[7] == ' ' || cp[7] == '\n')
 				) {
@@ -232,7 +232,7 @@ int split_merge(struct stream f, struct stream *f1, struct stream *f2, struct st
 				copyline(&r1, &cp, end);
 			break;
 		case 2:
-			if (len>8 &&
+			if (len>=8 &&
 			    strncmp(cp, "=======", 7)==0 &&
 			    (cp[7] == ' ' || cp[7] == '\n')
 				) {
@@ -242,7 +242,7 @@ int split_merge(struct stream f, struct stream *f1, struct stream *f2, struct st
 				copyline(&r2, &cp, end);
 			break;
 		case 3:
-			if (len>8 &&
+			if (len>=8 &&
 			    strncmp(cp, ">>>>>>>", 7)==0 &&
 			    (cp[7] == ' ' || cp[7] == '\n')
 				) {
diff --git a/merge.c b/merge.c
index 1196b85..ef222b7 100644
--- a/merge.c
+++ b/merge.c
@@ -422,6 +422,15 @@ static int advance(struct csl *c1, struct csl *c2, struct point *p)
 		if (c1[p->c1].len == 0 ||
 		    a < c1[p->c1].a + c1[p->c1].len) {
 			p->in_a = 0;
+			/*
+			 * if we've slid, make sure not to skip over
+			 * the stuff in c2.
+			 */
+			if(slid && p->c2 != -1 && c2[p->c2].a == b && 
+			   c2[p->c2].b > c2[p->c2].a) {
+				c -= c2[p->c2].b - c2[p->c2].a;
+			}
+
 			p->pos = c;
 			slid = 1;
 			goto retry;
-- 
1.7.0

